src/components/lyne-teaser/lyne-teaser.tsx

```diff
@@ -78,9 +85,21 @@ export class LyneTeaser {
    */
   @Prop() public pictureSizesConfig?: string;
 
+  /** Host element */
+  @Element() private _hostElement: HTMLElement;
+
+  private _hasImageSlot: boolean;
+  private _hasHeadlineSlot: boolean;
+  private _hasDescriptionSlot: boolean;
+
   public componentWillLoad(): void {
     // Validate props
     this.validateAccessibilityLabel(this.accessibilityLabel);
+
+    // Check slots
+    this._hasImageSlot = Boolean(this._hostElement.querySelector('[slot="image"]'));
```

Comment:
Sorry to explain this after the fact; Only checking whether the slots are used once on creation is bad practice. I'm aware that this pattern is used everywhere in existing components, which is something we will need to fix.
Please always render slots and be aware that content might be added dynamically.